include/crm/common/xml_internal.h

```diff
@@ -425,6 +425,19 @@ xmlNode *pcmk__xe_next_same(const xmlNode *node);
 void pcmk__xe_set_content(xmlNode *node, const char *format, ...)
     G_GNUC_PRINTF(2, 3);
 
+/*!
+ * \internal
+ * \enum pcmk__xa_copy_flags
+ * \brief Flags for copying XML attributes
+ */
+enum pcmk__xa_copy_flags {
+    //! Flag has no effect
+    pcmk__xa_copy_none          = 0U,
+
+    //! Don't overwrite existing values
+    pcmk__xa_copy_no_overwrite  = (1U << 0),
+};
```

Comment:
Since this is likely to see more use, s/xa_copy/xaf/

Reply:
They're not flags for attributes (which sort of suggests that they apply to the attrs themselves), they're specifically flags for copying/updating attributes. The most general we could say is that they're flags for attr operations.

`xaf` makes the lengths more manageable so I'm not gonna fight about it either way.

Reply:
could do something like `xa_op_*`... it's internal and I'm not that worried about current choice, but `xaf` does come out very close to `xml_private_flags`: `pcmk__xf_*`.

going with `xaf` for now, just a thought

Reply:
maybe `pcmk__xav_*` (for XML attribute value)